### PR TITLE
Moath/at 96/entry pages

### PR DESCRIPTION
**Pull request by mfalthaw**

### CODE REVIEW

File: src/view/components/AddEntry/index.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    /** Init state */
+    this.state = {
+      amountOfProduct: 0,
+      quality: '0',
+      costPerUnit: 0,
+    };
+  }
+
+  /** Get current datetime in specified format */
+  private getDatetime = (format: string) => moment().format(format);
```
Comment (jeinglis) [inline src/view/components/AddEntry/index.tsx]: I think you should use .local() to make sure it's the right time zone.

File: src/view/components/EditEntry/index.tsx [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+  // @TODO replace 'fakeFarmerUUID' with the active farmer uuid
+  const farmerRow = state.farmer.rows.find(r => r.uuid === 'fakeFarmerUUID');
+  if (farmerRow === undefined) {
+    throw new Error('Error: could not locate farmer: ' + 'fakeFarmerUUID');
+  }
+  // @TODO replace 'fakeMilkEntryUUID' with the active MilkEntry uuid    
+  const milkEntryRow = state.milk.rows.find(r => r.uuid === 'fakeMilkEntryUUID');
+  if (milkEntryRow === undefined) {
+    throw new Error('Error: could not locate MilkEntry: ' + 'fakeMilkEntryUUID');
+  }
```
Comment (jeinglis) [inline src/view/components/EditEntry/index.tsx]: do we want all these references to fake data in our master. Are we keeping it?

File: src/view/components/AddEntry/index.tsx [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+interface StorePropsType {
+  farmer: Farmer;
+}
+
+/** AddEntry PropsType */
+type PropsType = StorePropsType & DispatchPropsType & OwnPropsType;
+
+interface OwnStateType {
+  amountOfProduct: number;
+  quality: string;
```
Comment (jeinglis) [inline src/view/components/AddEntry/index.tsx]: I wonder if we should make this number since it's a measured value (I don't know the units though)

Comment (mfalthaw) [reply src/view/components/AddEntry/index.tsx]: Are we sure it's a number though?

File: src/view/components/EditEntry/index.tsx (hunk 2 above)
Comment (mfalthaw) [reply src/view/components/EditEntry/index.tsx]: We don't, you're right.

File: src/store/modules/activeRows/types.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
+/*----------------------- State -----------------------*/
+/** currentFarmer State*/
+export interface ActiveRowsState {
```
Comment (jeinglis) [inline src/store/modules/activeRows/types.ts]: why have you renamed it this? Do you plan to put all active info in this like the milk transaction, trader and farmer?

**Comment by mfalthaw** [comment]
closing this, new pr to right2drive/stitch

**Pull request closed without merging**